app/controllers/application_controller.rb

```diff
@@ -1,3 +1,13 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+
+  def receive
+    recieve = 'hogehogehogehoge'
+
+  end
+
+
+  def defualt
+    programmme = 'deualt'
```

Comment:
__[Misspell]__

"programmme" is a misspelling of "programme"


<!-- Comment by [Sider](https://sider.review) -->